### DISCUSSION TITLE
High precision mouse sensitivity is significantly different than windows sensitivity?

### DISCUSSION THREAD

**Discussion by madman404** [ppy/osu#30409] [Q&A] [answered]
Hello! I was troubleshooting some mouse tracking oddities, and as part of my testing, I found that I had high precision mouse disabled for reasons I did not remember. I turned it back on, and it seems that my sensitivity when it is on is _significantly_ lower than in the rest of windows even when sensitivity is set to 1x, making it nearly unusable. In my windows settings, I use 6/11 without enhance pointer precision, which should be no acceleration.

Does high precision mouse sensitivity on osu operate on an entirely different scale to unscaled sensitivity in windows? If it provides any further context, I play on a 4K monitor.

**Comment by peppy** [comment, marked as answer]
High precision bypasses windows sensitivity/acceleration completely. If you want it to match windows, please leave this setting off.

If you want to use the setting, change the in-game value until it matches what you want.

**Reply by madman404** [reply]
Is there any calculation I could do to match windows as a baseline, or is that something I'd just have to measure by hand? If I can calculate it, I could probably set a value that would give me the same ingame sensitivity between windows and linux, which would be nice. If not, no problem, I'll just leave it off.

**Reply by peppy** [reply]
Trial and error should be fast enough.
